packages/AlgernonTests.package/ALGTestAction.class/instance/core..st

```diff
@@ -0,0 +1,3 @@
+accessing
+core: anObject
+	core := anObject
```

Comment:
These can also be deleted see `testDoIt`

Reply:
not really, this one is called by other methods